README.md

```diff
@@ -22,17 +22,19 @@ Outer for loop, inner while loop, and merge.
 Since the merge function contains a in essence a loop we can analyze the worst case scenerio
 by using loop logic. Ex: a loop inside a loop is equivilant to n*n.
 
-The outer for loop runs while size < the array length, and the inner while loop runs when j < the array length.
-Since size is doubled each iteration and j is incremented by size, these loops are reminicent of an exponential,
-which results in log(n) run time.
+The outer for loop runs while size < the array length and since size is doubled each iteration and this loop 
+is reminicent of an exponential, which results in log(n) run time.
+
+The inner while loop run while j < the array length and will happen length/2*size times,
+which results in n runtime.
 
 The final piece is merge which has 2 loops. The outer while loop runs while i is less then j or
 essentially n/size times as it runs once for each array needing to be "merged". As we are analyzing the worst case
 scenerio, the inner loop will run essentially every time as no elements are sorted in our array. This results
 in the inner for loop running as long as k <= size + j - 1 and since k is incremented by 1 each time, it will run
 size + j - 1 times. Size and j are both elements of O(n) so this loop will run O(n) times in the wost case.
 
-Now we multiply our run times together to get the wost case which results in $\Theta (n^2log(n))$ as our
+Now we multiply our run times together to get the wost case which results in $\Theta (n^3log(n))$ as our
```

Comment:
Based on your text above, I would expect to see $n\log{n}$ here. Don't know where the $n^3$ comes from.

Reply:
I don't know if this makes more sense, but I'm multiplying the nlog(n) with n^2 as the nlog(n) comes from the outer loops and the n^2 comes from the merge function itself. That's how I'm getting the n^3log(n)